tests/webhook/webhook_test.py

```diff
@@ -141,10 +142,12 @@ def test_handle_org_event_rm_member_missing(mock_logging, org_rm_payload):
     mock_facade = mock.MagicMock(DBFacade)
     mock_facade.query_user.return_value = []
     webhook_handler = WebhookHandler(mock_facade)
-    webhook_handler.handle_organization_event(org_rm_payload)
+    rsp, code = webhook_handler.handle_organization_event(org_rm_payload)
     mock_facade.query_user\
-        .assert_called_once_with(['github_id', 39652351])
+        .assert_called_once_with([('github_id', 39652351)])

```

Comment:
wait why facade returns an array of tuples now? I thought query user only queries one user at the moment

Reply:
It doesn't, `query_user()` returns a list of users given a list of tuples, where each tuple is an attribute and value